### ISSUE TITLE
Randomized videos

### ISSUE THREAD

**Issue by SimplyCorbett** [Chocobozzz/PeerTube#4406]
**Describe the problem to be solved**

Videos on recently added and local videos show videos in the same order uploaded without an option to truly randomize videos.

**Describe the solution you would like:**

Add a "random" category/option that will randomize videos displayed instead of showing the videos in the order uploaded.

**Comment by Chocobozzz** [comment]
Hello,

Random sort is difficult to execute in the database and I don't think it would be very useful since we already have random data in discover section.

**Comment by SimplyCorbett** [comment]
My discover page has "no results". I thought it was for federation only since no local videos were showing up in it and I don't use federation.

So randomized won't be possible? bummer.

**Comment by Chocobozzz** [comment]
What is your instance URL? You need to have some videos on your instance so peertube tries to create sample data

**Comment by SimplyCorbett** [comment]
It's a LAN only instance. Thinking it over the problem is probably because the videos are internal only. There are ~650 videos though.

**Comment by Chocobozzz** [comment]
Yes you're right, the discover page only displays public videos.
